cmd/mcp/README.md

```diff
@@ -0,0 +1,271 @@
+  - `limit`: The maximum number of resources to return (optional).
+- `get_kubernetes_logs` : This tool retrieves the most recent logs of a Kubernetes pod.
+  - `pod_name` - The name of the pod (required).
+  - `pod_namespace` - The namespace of the pod (required).
+  - `containe_namer` - The name of the container (required).
+  - `limit` - The maximum number of lines to return (default 100).
+- `get_kubernetes_api_versions`: This tool retrieves the CRDs registered on the cluster and returns the preferred apiVersion for each kind.
+  - No arguments required
+
+The output of the reporting tools is formatted as a multi-doc YAML.
+
+### Reconciliation tools
+
+The MCP server provides a set of tools for triggering the reconciliation of Flux resources:
+
+- `reconcile_flux_resourceset`: This tool triggers the reconciliation of the Flux ResourceSet.
+  - `name` - The name of the resource to reconcile (required).
+  - `namespace` - The namespace of the resource to reconcile (required).
+- `reconcile_flux_source`: This tool triggers the reconciliation of the Flux sources (GitRepository, OCIRepository, HelmRepository, HelmChart, Bucket).
+  - `name` - The name of the resource to reconcile (required).
+  - `namespace` - The namespace of the resource to reconcile (required).
+- `reconcile_flux_kustomization`: This tool triggers the reconciliation of the Flux Kustomization including its source (GitRepository, OCIRepository, Bucket).
+  - `name` - The name of the resource to reconcile (required).
+  - `namespace` - The namespace of the resource to reconcile (required).
+  - `with_source` - Trigger the reconciliation of the Flux Kustomization source (optional).
+- `reconcile_flux_helmrelease`: This tool triggers the reconciliation of the Flux HelmRelease including its source (OCIRepository, HelmChart).
+  - `name` - The name of the resource to reconcile (required).
+  - `namespace` - The namespace of the resource to reconcile (required).
+  - `with_source` - Trigger the reconciliation of the Flux HelmRelease source (optional).
+
+The output of the reconciliation tools tells the model how to verify the status of the reconciled resource.
+
+### Suspend / Resume tools
+
+The MCP server provides a set of tools for suspending and resuming the reconciliation of Flux resources:
+
+- `suspend_flux_resource`: This tool suspends the reconciliation of a Flux resource (Kustomization, HelmRelease, ResourceSet, OCIRepository, etc.).
+  - `apiVersion` - The API version of the resource (required).
+  - `kind` - The kind of the resource (required).
+  - `name` - The name of the resource (required).
+  - `namespace` - The namespace of the resource (required).
+- `resume_flux_resource`: This tool resumes the reconciliation of a Flux resource.
+  - `apiVersion` - The API version of the resource (required).
+  - `kind` - The kind of the resource (required).
+  - `name` - The name of the resource (required).
+  - `namespace` - The namespace of the resource (required).
```

Comment:
Looks like these two are now called `suspend_flux_reconciliation` and `resume_flux_reconciliation`